app/public/css/styles.css

```diff
@@ -3025,7 +3034,18 @@ button[data-balloon] {
   }
 
   .export-content .button {
-    width: 100%;
+    flex: 1;
+    white-space: nowrap;
+    padding: 10px 15px;
+  }
+
+  .export-content-bottom {
+    margin-bottom: 20px;
```

Comment:
I think the general convention is to use multiples of 8, can you confirm @d-novak ?

Reply:
inspired by   
.contact-form-bottom {
    margin-bottom: 20px;
    align-items: stretch;
  },

Reply:
Yes, the spacing convention back when I implemented the redesign was that we use only values dividable by 8, in edge cases dividable by 4 (like when you need something between `0 - 8px`). I haven't seen the new designs for the contact form (modal) but the spacing there should also be `8 * X`, thanks for pointing that out. I think the best idea is to leave the `.contact-form-bottom` as it is for now, @ladomyriak is implementing this part in another PR so maybe he can take a look at the spacings there as well.